Documentation/cross-building.md

```diff
@@ -0,0 +1,76 @@
+Cross Compilation for ARM on Linux
+==================================
+
+Through cross compilation, on Linux it is possible to build CoreCLR for arm or arm64.
+
+Requirements
+------------
+
+You need a Debian based host and the following packages needs to be installed:
+
+    $ sudo apt-get install qemu qemu-user-static binfmt-support debootstrap
+
+In addition, to cross compile CoreCLR the binutils for the target are required. So for arm you need:
+
```

Comment:
Just a note that I added one commit which changes the meaning of ROOTFS_DIR. I'm also going to do a pull request for CoreCLR to match this new meaning.